src/docs/user/references/VariablesReference.adoc

```diff
@@ -1,140 +1,183 @@
+| *Job ID*. ID of the current job.
 | variables.get( "PA_JOB_ID" )
 | $variables_PA_JOB_ID
-|
-${PA_JOB_ID}
+| ${PA_JOB_ID}
+| all scripts
 
-| *Job name*. Available via script task, pre script, post script, flow script.
+| *Job name*. Name of the current job.
 | variables.get( "PA_JOB_NAME" )
 | $variables_PA_JOB_NAME
 | ${PA_JOB_NAME}
+| all scripts
 
-| *Task ID*. Available via script task, pre script, post script, flow script.
+| *Task ID*. Id of the task.
 | variables.get( "PA_TASK_ID" )
 | $variables_PA_TASK_ID
 | ${PA_TASK_ID}
+| all scripts
 
-| *Task name*. Available via script task, pre script, post script, flow script.
+| *Task name*. Name of the task.
 | variables.get( "PA_TASK_NAME" )
 | $variables_PA_TASK_NAME
 | ${PA_TASK_NAME}
+| all scripts
 
-| *Task iteration index*. See <<_task_executable>>. Available via script task, pre script, post script, flow script.
+| *User*. The name of user which submitted the job.
+| variables.get( "PA_USER" )
+| $variables_PA_USER
+| ${PA_USER}
+| all scripts
+
+| *Scheduler Home*. Installation directory of the scheduler on the ProActive node.
+| variables.get( "PA_SCHEDULER_HOME" )
+| $variables_PA_SCHEDULER_HOME
+| -
+| all scripts except <<_selection, selection script>>
+
+| *Task iteration index*. Index of iteration for this task inside a loop control. See <<_task_executable>>.
 | variables.get( "PA_TASK_ITERATION" )
 | $variables_PA_TASK_ITERATION
 | ${PA_TASK_ITERATION}
+| all scripts
 
-| *Task replication index*. See <<_task_executable>>. Available via script task, pre script, post script, flow script.
+| *Task replication index*. Index of replication for this task inside a replicate control. See <<_task_executable>>.
 | variables.get( "PA_TASK_REPLICATION" )
 | $variables_PA_TASK_REPLICATION
 | ${PA_TASK_REPLICATION}
+| all scripts
+
+| *Task Success*. A variable accessible only from the clean script. It is a boolean value which is _true_ is the task succeeded and _false_ if the task failed.
+| variables.get( "PA_TASK_SUCCESS" )
+| $variables_PA_TASK_SUCCESS
+| -
+| <<_pre_post_clean, clean script>>
 
-| *Third party credentials*. Available via script task.
+| *Third party credentials*. Credentials stored on the server for this user account
 | credentials.get( "pw" )
 | -
 | $credentials_pw (only in the task arguments)
+| <<_pre_post_clean, pre/post>> (not available on clean script),<<_scripting_language_support, script task>>,<<_fork_environment, environment>>,<<_control_flow_scripts,flow script>>
 
-| *SSH private key*. See <<_run_computation_with_your_system_account>>. Available via script task.
+| *SSH private key*. Private SSH Key used at login. See <<_run_computation_with_your_system_account>>.
 | credentials.get( "SSH_PRIVATE_KEY" )
 | -
 | -
+| <<_pre_post_clean, pre/post>> (not available on clean script),<<_scripting_language_support, script task>>,<<_fork_environment, environment>>,<<_control_flow_scripts,flow script>>
```

Comment:
Would be great to add a space after all comma to improve readability. It applies to all other rows where the same text has been copy/paste.